lib/game_modules/game.ex

```diff
@@ -1,12 +1,23 @@
 defmodule Game do
 
-  def place_move(position, board, icon) do
-    position_key = Board.position_to_key_atom(position)
-    Map.put(board, position_key, icon)
+  def mark_square(board, row, column, icon) do
+    old_row = MatrixBoard.get_row_at(board, row)
+    new_row = List.replace_at(old_row, column, icon)
+    List.replace_at(board, row, new_row)
   end
 
-  # def won? do
-  #   false
-  # end
+  def has_win?(board) do
+    MatrixBoard.has_winning_row?(board) ||
```

Comment:
Probably OK to name this `Board` and delete the old implementation if you're going to go with this one. Don't worry, the old `Board` is in git if you ever decide you need it again!